{
  "uid" : "8ba706878273a791",
  "name" : "User can't successfully log in with invalid password.",
  "fullName" : "tests.core_features.test_login.TestLogin#test_login_with_invalid_password",
  "historyId" : "0ca18d8bd97e96e2fad3ec13e04117a8",
  "time" : {
    "start" : 1705147631883,
    "stop" : 1705147648784,
    "duration" : 16901
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.TimeoutException: Message:",
  "statusTrace" : "tests/core_features/test_login.py:50: in test_login_with_invalid_password\n    login_page.error_text.assert_error(\"The username and password could not be verified.\")\n        driver     = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"02e402f087c4e2b0addf65e35ddc2fc4\")>\n        home_page  = <src.pages.home_page.HomePage object at 0x7fdbd5482410>\n        login_page = <src.pages.login_page.LoginPage object at 0x7fdbd54df110>\n        self       = <test_login.TestLogin object at 0x7fdbd3b934d0>\nsrc/base/base_element.py:69: in assert_error\n    current_error = self.get_text()\n        expected_error = 'The username and password could not be verified.'\n        self       = <src.base.base_element.BaseElement object at 0x7fdbd54804d0>\n        server_error = 'An internal error has occurred and has been logged.'\nsrc/base/base_element.py:37: in get_text\n    element = self.assert_element()\n        self       = <src.base.base_element.BaseElement object at 0x7fdbd54804d0>\nsrc/base/base_element.py:16: in assert_element\n    self.wait.until(EC.presence_of_element_located((\"xpath\", self.xpath)))\n        clickable  = False\n        return_many = False\n        self       = <src.base.base_element.BaseElement object at 0x7fdbd54804d0>\n../local/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: in until\n    raise TimeoutException(message, screen, stacktrace)\nE   selenium.common.exceptions.TimeoutException: Message:\n        end_time   = 303.007895139\n        message    = ''\n        method     = <function presence_of_element_located.<locals>._predicate at 0x7fdbd6c21440>\n        screen     = None\n        self       = <selenium.webdriver.support.wait.WebDriverWait (session=\"02e402f087c4e2b0addf65e35ddc2fc4\")>\n        stacktrace = None",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "Health check request",
    "time" : {
      "start" : 1705147517847,
      "stop" : 1705147518594,
      "duration" : 747
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1705147517826,
      "stop" : 1705147517847,
      "duration" : 21
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "Initialize driver",
    "time" : {
      "start" : 1705147631046,
      "stop" : 1705147631882,
      "duration" : 836
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.TimeoutException: Message:",
    "statusTrace" : "tests/core_features/test_login.py:50: in test_login_with_invalid_password\n    login_page.error_text.assert_error(\"The username and password could not be verified.\")\n        driver     = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"02e402f087c4e2b0addf65e35ddc2fc4\")>\n        home_page  = <src.pages.home_page.HomePage object at 0x7fdbd5482410>\n        login_page = <src.pages.login_page.LoginPage object at 0x7fdbd54df110>\n        self       = <test_login.TestLogin object at 0x7fdbd3b934d0>\nsrc/base/base_element.py:69: in assert_error\n    current_error = self.get_text()\n        expected_error = 'The username and password could not be verified.'\n        self       = <src.base.base_element.BaseElement object at 0x7fdbd54804d0>\n        server_error = 'An internal error has occurred and has been logged.'\nsrc/base/base_element.py:37: in get_text\n    element = self.assert_element()\n        self       = <src.base.base_element.BaseElement object at 0x7fdbd54804d0>\nsrc/base/base_element.py:16: in assert_element\n    self.wait.until(EC.presence_of_element_located((\"xpath\", self.xpath)))\n        clickable  = False\n        return_many = False\n        self       = <src.base.base_element.BaseElement object at 0x7fdbd54804d0>\n../local/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:101: in until\n    raise TimeoutException(message, screen, stacktrace)\nE   selenium.common.exceptions.TimeoutException: Message:\n        end_time   = 303.007895139\n        message    = ''\n        method     = <function presence_of_element_located.<locals>._predicate at 0x7fdbd6c21440>\n        screen     = None\n        self       = <selenium.webdriver.support.wait.WebDriverWait (session=\"02e402f087c4e2b0addf65e35ddc2fc4\")>\n        stacktrace = None",
    "steps" : [ {
      "name" : "Navigate to Home page",
      "time" : {
        "start" : 1705147631883,
        "stop" : 1705147632540,
        "duration" : 657
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Fill in valid username, but invalid password",
      "time" : {
        "start" : 1705147632540,
        "stop" : 1705147632698,
        "duration" : 158
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Log in",
      "time" : {
        "start" : 1705147632698,
        "stop" : 1705147633581,
        "duration" : 883
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    }, {
      "name" : "Check error message is displayed",
      "time" : {
        "start" : 1705147633581,
        "stop" : 1705147648784,
        "duration" : 15203
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: \n\n",
      "statusTrace" : "  File \"/usr/workspace/tests/core_features/test_login.py\", line 50, in test_login_with_invalid_password\n    login_page.error_text.assert_error(\"The username and password could not be verified.\")\n  File \"/usr/workspace/src/base/base_element.py\", line 69, in assert_error\n    current_error = self.get_text()\n                    ^^^^^^^^^^^^^^^\n  File \"/usr/workspace/src/base/base_element.py\", line 37, in get_text\n    element = self.assert_element()\n              ^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/workspace/src/base/base_element.py\", line 16, in assert_element\n    self.wait.until(EC.presence_of_element_located((\"xpath\", self.xpath)))\n  File \"/usr/local/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 101, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 4,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "Initialize driver::0",
    "time" : {
      "start" : 1705147648792,
      "stop" : 1705147648851,
      "duration" : 59
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Terminate driver",
      "time" : {
        "start" : 1705147648792,
        "stop" : 1705147648851,
        "duration" : 59
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : false
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "PARA-15240: User registration and authentication"
  }, {
    "name" : "feature",
    "value" : "PARA-15261: User login"
  }, {
    "name" : "severity",
    "value" : "Minor"
  }, {
    "name" : "parentSuite",
    "value" : "tests.core_features"
  }, {
    "name" : "suite",
    "value" : "test_login"
  }, {
    "name" : "subSuite",
    "value" : "TestLogin"
  }, {
    "name" : "host",
    "value" : "fdd3900bcbbf"
  }, {
    "name" : "thread",
    "value" : "1-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.core_features.test_login"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "8ba706878273a791.json",
  "parameterValues" : [ ]
}